### PR TITLE
PRMT-4738

### PR DESCRIPTION
**Pull request by martin-nhs**
- Delegated update status to axios callback.

### CODE REVIEW

File: src/services/transfer/transfer-out-ehr-core.js [hunk 1]
```diff
@@ -54,13 +54,6 @@ export async function transferOutEhrCore({ conversationId, nhsNumber, odsCode, e
...
     );
-
-    await updateConversationStatus(
-      conversationId,
-      ConversationStatus.OUTBOUND_SENT_CORE,
-      null,
-      'The EHR Core has successfully been sent.'
-    );
   } catch (error) {
     await handleCoreTransferError(error, conversationId);
```
Comment (chrisbloe-nhse) [inline src/services/transfer/transfer-out-ehr-core.js:58]: in/out?

File: src/services/transfer/transfer-out-util.js [hunk 2]
```diff
@@ -59,6 +60,21 @@ export const updateFragmentStatus = async (
   logInfo(`Updated fragment status to: ${status}`);
 };
 
+export const updateCoreStatus = async (
+    outboundConversationId,
+    status,
+    failureReason = null
+) => {
+  setCurrentSpanAttributes({ conversationId: outboundConversationId });
```
Comment (chrisbloe-nhse) [inline src/services/transfer/transfer-out-util.js:68]: in/out?

File: src/services/gp2gp/send-core.js [hunk 3]
```diff
@@ -3,6 +3,8 @@ import { logInfo } from '../../middleware/logging';
 import { config } from '../../config';
 import { SendCoreError } from "../../errors/errors";
 import { logOutboundMessage } from "./logging-utils";
+import { updateConversationStatus, updateCoreStatus } from "../transfer/transfer-out-util";
+import { ConversationStatus, CoreStatus } from "../../constants/enums";
 
 export const sendCore = async (conversationId, odsCode, coreEhr, ehrRequestId, newMessageId) => {
```
Comment (chrisbloe-nhse) [inline src/services/gp2gp/send-core.js:9]: in/out? (x4)

File: src/services/transfer/transfer-out-util.js (hunk 2 above)
Comment (martin-nhs) [reply src/services/transfer/transfer-out-util.js:68]: Addressed

File: src/services/gp2gp/send-core.js (hunk 3 above)
Comment (martin-nhs) [reply src/services/gp2gp/send-core.js]: Addressed

File: src/models/fragment.js [hunk 4]
```diff
@@ -1,8 +1,7 @@
 import { FragmentStatus, RecordType } from '../constants/enums';
-import { validate } from 'uuid';
 import { getUKTimestamp } from '../services/time';
 import { addChangesToUpdateParams } from '../utilities/dynamodb-helper';
-import { ValidationError } from '../errors/errors';
+import { validateIds } from "../utilities/validation-utilities";
```
Comment (chrisbloe-nhse) [inline src/models/fragment.js:4]: Double quote?

Comment (martin-nhs) [reply src/models/fragment.js]: Addressed

**Review by MohammadIqbalAD-NHS — APPROVED** [review]
Looks good!

**Pull request merged**